library-verification.toml

```diff
@@ -0,0 +1,12 @@
+# libnvme is a global zone only library and therefore we must be sure that only
+# programs running in the gz require it. Additionally only sled-agent should be
+# managing a sled's hardware.
+[libraries."libnvme.so.1"]
+binaries = [
```

Comment:
So it isn't clear that this is actually an allowlist -- it could just as well be a denylist.

Could this be made clearer, e.g. by calling this `allowlist-binaries` or just `allowlist`?